api_definition.go

```diff
@@ -205,8 +205,8 @@ func (a *APIDefinitionLoader) MakeSpec(appConfig *apidef.APIDefinition) *APISpec
 		}
 	}
 
-	newAppSpec.RxPaths = make(map[string][]URLSpec)
-	newAppSpec.WhiteListEnabled = make(map[string]bool)
+	newAppSpec.RxPaths = make(map[string][]URLSpec, len(appConfig.VersionData.Versions))
```

Comment:
> You may call it premature optimization
I do :)

It is non-default behavior and usually not used for maps. That's why I was wondering, and someone else who will work on this code can too.

If you really want to keep them here, I'm ok, but it just attracts attention.